docker-java-api/src/main/java/com/github/dockerjava/api/command/ListImagesCmd.java

```diff
@@ -44,6 +44,13 @@ public interface ListImagesCmd extends SyncDockerCmd<List<Image>> {
      */
     ListImagesCmd withLabelFilter(Map<String, String> labels);
 
+    /**
+     * Filter images by reference
+     *
+     * @param reference string in the form {@code <image-name>[:<tag>]}
+     */
+    ListImagesCmd withReferenceFilter(String reference);
```

Comment:
Thinking out loud: should we add `withFilter(String name, Collection<String> value)` and make these methods `default` implementations that would call it?

Reply:
agree!

Reply:
This is a good point to not break existing implementations. However, `FiltersBuilder` belongs to the `core` module, not the `api`